email_templates/approvedEmail/html.ejs

```diff
@@ -0,0 +1,12 @@
+<html>
```

Comment:
Feel free to edit these files with whatever you want the emails to say. They're just templates.